src/vaadin-number-field.html

```diff
@@ -418,6 +418,21 @@
 
           return super.checkValidity();
         }
+
+        /**
+          * Native [type=number] inputs don't update their value
+          * when you are entering input that the browser is unable to parse
+          * e.g. "--5", hence we have to override this method from `InputMixin`
+          * so that, when value is empty, it would additionally check
+          * for bad input based on the native `validity.badInput` property.
+          *
+          * @param {InputEvent} event
+          * @protected
+          * @override
+          */
```

Comment:
```suggestion
         * Native [type=number] inputs don't update their value
         * when you are entering input that the browser is unable to parse
         * e.g. "--5", hence we have to override this method from `InputMixin`
         * so that, when value is empty, it would additionally check
         * for bad input based on the native `validity.badInput` property.
         *
         * @param {InputEvent} event
         * @protected
         * @override
         */
```